package.json

```diff
@@ -35,7 +35,7 @@
     "gelf-stream": "^1.1.1",
     "hot-shots": "^4.3.1",
     "js-yaml": "^3.8.1",
-    "limitation": "^0.1.9",
+    "limitation": "git+https://github.com/arlolra/limitation#cleanup",
```

Comment:
I will review the `limitation PR`, we'll bump a version there and replace this with normal dependency before merging.